src/material/select/select.scss

```diff
@@ -72,6 +72,13 @@ $mat-select-placeholder-arrow-space: 2 * ($mat-select-arrow-size + $mat-select-a
   ._mat-animation-noopable.mat-form-field-appearance-standard .mat-select.mat-select-empty & {
     transition: none;
   }
+
+  // On Safari the arrow won't necessarily get redrawn when the form field is focused or blurred,
+  // which means that its highlighted styling will be stuck in an incorrect state. We can get
+  // Safari to redraw it by adding a transform.
+  .mat-form-field.mat-focused & {
+    transform: translateX(0);
```

Comment:
This will override the above styles which are applying `transform: translateY`. Is there another way that can force Safari to redraw?

Reply:
I've moved the `transform` to a different element.